src/V3EmitCSyms.cpp

```diff
@@ -467,7 +468,7 @@ void EmitCSyms::emitSymHdr() {
 
     puts("\n// CREATORS\n");
     puts(symClassName() + "(" + topClassName() + "* topp, const char* namep);\n");
-    puts(string("~") + symClassName() + "() = default;\n");
+    puts(string("~") + symClassName() + "();\n");
```

Comment:
OK, to always do this?  Or should I make this conditional on VPI?

Reply:
Always seems fine, not fast path.   Someday this will be better abstracted and it'll optimize to this if empty.